CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Minio cookbook contribution guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio cookbook is the collection of community generated detailed guides to set up Minio with other tools/platforms. If you have a 
+working setup that uses Minio, we welcome you to add your guide to the Cookbook. Please add the below setions in your guide before adding a 
```

Comment:
Minio cookbook is not just the collection of community guides for using Minio with their tools / platforms. Minio cookbook should not come across as these are not officially syndicated by Minio. Also lot of cookbook content ins generated by Minio team. We encourage community to contribute here but this is not exclusively community generated.